.github/workflows/main.yml

```diff
@@ -0,0 +1,29 @@
+name: noop run
+
+on: pull_request
+
+jobs:
+  unit:
+    runs-on: ubuntu-latest
+    name: Run msync --noop against all modules
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup ruby
+        uses: actions/setup-ruby@v1
+        with:
+          ruby-version: 2.7
+      - uses: actions/cache@v2
+        with:
+          path: vendor/bundle
+          key: ${{ runner.os }}-gems-${{ hashFiles('**/Gemfile.lock') }}
```

Comment:
I'm still not sure if I need to include the Ruby version or not if it's not a matrix. Thoughts?

Reply:
I think you could test this by:

1. Push a commit with empty `.github/workflows/main.yml` directly to master in order to get CI running
2. Push this branch with a 2nd commit that has some dummy change that would be reflected in `bundle exec msync update --noop`
3. When satisfied with how CI is running, remove the 2nd commit and force push

Reply:
That's not really the point. It's more that the cache probably isn't invalidated if you later upgrade to Ruby 3.0.

Reply:
> Push a commit with empty `.github/workflows/main.yml` directly to master in order to get CI running

Also, in this PR I had a better workaround that appears to work. First create a branch (I created `gha-test`) which includes the skeleton. That triggers a build. Then you switch the base branch to master without the skeleton. That keeps the git log clean while it still tests.